src/popup.js

```diff
@@ -508,7 +508,7 @@ function setTabToUrl( query_url ) { /* jshint ignore:line */
 }
 
 document.addEventListener('DOMContentLoaded', function () {
-  chrome.tabs.query({active: true}, function(t) {
+  chrome.tabs.query({active: true, currentWindow: true}, function(t) {
```

Comment:
@cooperq this fixes the bug with the tab datastructure you saw. If you have two separate non-incognito windows it comes up too.